src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -223,7 +223,10 @@ CommandsRegistry.registerCommand({
 
 		for (let editor of editors) {
 			if (editor instanceof NotebookInput) {
-				let model: INotebookModel = editor.notebookModel;
+				// Wait for NotebookInput.resolve() to be called to ensure the _model field in NotebookInput
+				// is not undefined when trying to get the notebookModel.
+				await editor.modelResolved;
+				let model: INotebookModel = (<NotebookInput>editor).notebookModel;
```

Comment:
Should be able to get rid of the cast here as well (it's already checked on line 225)